addons/libs/images.lua

```diff
@@ -407,6 +426,9 @@ windower.register_event('mouse', function(type, x, y, delta, blocked)
             if meta[dragged.image].root_settings then
                 config.save(meta[dragged.image].root_settings)
             end
+            if x == clicked.x and y == clicked.y then
+                call_events(dragged.image, 'left_click')
```

Comment:
I was debating whether we should leave it as `left_click` or just call it `click` and have `right_click` as the odd one out, because it's used more rarely. I'm not sure though, would be up for discussion. Could also alias it so `click` and `left_click` are identical in behavior.

Reply:
I should be able to easily add support for right click, too. Would it be silly to assume that type 3 = right click and type 4 = right release? I would prefer adding that functionality while we're at it, for the community's sake.

Reply:
> Please advise the types (integer) for the right click/release events. (I don't see them documented anywhere.)

No idea, easy for you to test. Just `windower.register_event('mouse', print)`, then right click. For reference, it's the `wParam` parameter of [`MouseProc`](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/ms644988(v=vs.85)), `& 0xFF`. Whatever that is.

> Please advise how I would setup the alias, if you decide that would be desired.

Actually, I'm tending towards leaving it as `left_click` here, but also handling `click` by looking up a mapping table when registering it. Can define a table like this:
```lua
local event_map = {
    click = 'left_click',
    double_click = 'double_left_click',
}
```

Then use this when registering the event, at the very top of the function:
```lua
    key = event_map[key] or key
```